docker-compose.yml

```diff
@@ -2,25 +2,21 @@ services:
 
   web:
     build:
-      context: ./ccm_web/
-      dockerfile: ./Dockerfile
+      context: .
+      dockerfile: Dockerfile
       args:
         - PORT=4000
-      target: base
-    working_dir: /base/
     environment:
       - PORT=4000
-      - NODE_ENV=development
-    env_file: .env
+      - DEBUG=True
```

Comment:
I found out with ROHQ that you can use this pattern to specify defaults in the docker-compose file

`      - DEBUG=${DEBUG:-True}`

I'd switch to this here for everything in the environment section. 

https://github.com/tl-its-umich-edu/remote-office-hours-queue/blob/e329460a143c18fdcaf3dbb24284879a30ebc9c2/docker-compose.yaml#L9